src/i18n.js

```diff
@@ -26,15 +23,18 @@ function loadLocaleConfig() {
   return messages;
 }
 
-const i18n = new VueI18n({
+const i18n = createI18n({
+  legacy: true,
+  globalInjection: true,
   locale: CONFIG.locale,
   fallbackLocale: CONFIG.fallbackLocale,
   messages: loadLocaleConfig(),
-  // Todo: Workaround for https://github.com/kazupon/vue-i18n/issues/563
+  // Workaround for https://github.com/kazupon/vue-i18n/issues/563
```

Comment:
Is this not a todo anymore? We should check what's the status of this in the new version